types/react/test/index.ts

```diff
@@ -11,7 +11,6 @@ import * as TestUtils from "react-addons-test-utils";
 import TransitionGroup = require("react-addons-transition-group");
 import update = require("react-addons-update");
 import createReactClass = require("create-react-class");
-import * as PropTypes from "prop-types";
```

Comment:
Hmm. The non-proptypes only test bits that rely in the proptypes types I'd leave in, if only to test the whole react (and it's expected usage) stack more thoroughly.

Reply:
I had to remove prop types as a dependency here to prevent the publisher from yelling at me (because React would then need to be updated to TS 2.8), so I removed the tests that used PropTypes exports.